tests/collections/client.test.ts

```diff
@@ -14,24 +14,28 @@
 
 import assert from 'assert';
 import { Client } from '@/src/collections/client';
-import { getAstraClient, astraUri } from '@/tests/fixtures';
+import { testClients } from '@/tests/fixtures';
 import { parseUri } from '@/src/collections/utils';
 import { AUTH_API_PATH } from '@/src/client/httpClient';
+import _ from 'lodash';
 
-
-describe('StargateMongoose - collections.Client', () => {
+for (const testClient in testClients) {
+describe('StargateMongoose clients test', () => {
```

Comment:
Minor nit: indent within for loop. In general, I think it might be cleaner to have a separate config option to run tests against Astra vs run tests against local, rather than having a for loop to run tests once for Astra and once for local. That way, we don't see a bunch of skipped tests in local dev.

Reply:
Makes sense. Let me change it to run tests individually instead of running in a loop.

Reply:
@vkarpov15 I changed the tests in [c303f3b](https://github.com/stargate/stargate-mongoose/pull/72/commits/c303f3bd2a51ebff1d2dafdb0c3c8ff54e2edbdc) to run them individually as required instead of running in a loop for Astra and JSON API

```
npm run test-astra
npm run test-jsonapi
`npm run test` which runs `npm run test-astra && npm run test-jsonapi`
```